049-cyrus-sasl-2.1.26-md5global.patch

Taken from upstream, should be staple in future official release.
diff -r 6675a2cd7300 -r a55ce431927e include/Makefile.am
--- a/include/Makefile.am	Thu Feb 02 08:34:32 2023 -0800
+++ b/include/Makefile.am	Thu Feb 02 08:36:12 2023 -0800
@@ -49,20 +49,7 @@
 
 noinst_PROGRAMS = makemd5
 
-makemd5_SOURCES = makemd5.c
-
-makemd5$(BUILD_EXEEXT) $(makemd5_OBJECTS): CC=$(CC_FOR_BUILD)
-makemd5$(BUILD_EXEEXT) $(makemd5_OBJECTS): CFLAGS=$(CFLAGS_FOR_BUILD)
-makemd5$(BUILD_EXEEXT): LDFLAGS=$(LDFLAGS_FOR_BUILD)
-
-md5global.h: makemd5$(BUILD_EXEEXT) Makefile
-	-rm -f $@
-	./$< $@
-
-BUILT_SOURCES = md5global.h
-
 EXTRA_DIST = NTMakefile
-DISTCLEANFILES = md5global.h
 
 if MACOSX
 framedir = /Library/Frameworks/SASL2.framework
